frontend/libs/studio-components/src/components/StudioCodelistEditor/StudioCodeListEditor.tsx

```diff
@@ -230,6 +224,26 @@ function Errors({ errorMap }: ErrorsProps): ReactElement {
   }
 }
 
+type TypeSelectorProps = {
+  setValueType: (valueType: CodeListItemValueLiteral) => void;
+};
+
+function TypeSelector({ setValueType }: TypeSelectorProps): ReactElement {
+  const { texts } = useStudioCodeListEditorContext();
+
+  return (
+    <StudioNativeSelect
+      className={classes.typeSelector}
+      label={texts.typeSelector.label}
+      onChange={(event) => setValueType(event.target.value as CodeListItemValueLiteral)}
```

Comment:
This could be moved to a `handleChange` function 😄

Reply:
I agree! Fixed.